### ISSUE TITLE
Integrate the new Circuit Breaker module

### ISSUE THREAD

**Issue by gjermundgaraba** [EmpowerPlastic/empowerchain#1009]
https://medium.com/the-interchain-foundation/building-resilience-in-blockchain-applications-with-the-circuit-breaker-module-5df0e0df930c

This work cannot start until a final release has been cut of the Circuit Breaker module. Looks like it will be soon-ish.

**Comment by ba1ciu** [comment]
Very low priority.
